src/lang/en-US.ts

```diff
@@ -420,7 +430,8 @@ const AdminPage: ResolvedIntlConfig['messages'] = {
 };
 
 const Welcome: ResolvedIntlConfig['messages'] = {
-    'welcome.image.alt': `content needed`,
+    'welcome.image.alt': `A diagram explaining Flow with endpoints coming into and out of ${CommonMessages.productName}.`,
```

Comment:
Suggestions to make these a bit more descriptive:

"A diagram showing the Flow logo at the center, connected by pipelines to multiple endpoint systems. Source systems on the left feed data into Flow and destination systems on the right receive data from Flow."

"A data pipeline diagram showing data moving into Flow from the Wikipedia HTTP source, and coming out of Flow into the Google Sheets destination."